readme.md

```diff
@@ -29,22 +29,22 @@ This can be achieved by:
 
 And both pin/lock to the same version.
 
-# npm@<7
+# Yarn
 
-`npm@<7` does not automatically install `peerDependencies`,
+Yarn does not automatically install `peerDependencies`,
 so if that's what you're using, install them manually.
 Here is an example, but use it only for reference,
 because your decisions regarding version ranges and range specifiers may vary.
 
-```
-npm install --save-dev \
+```sh
+yarn add \
   "typescript@*" \
   "eslint@^8.0.1" \
   "eslint-plugin-promise@^6.0.0" \
   "eslint-plugin-import@^2.25.2" \
   "eslint-plugin-n@^15.0.0" \
   "@typescript-eslint/eslint-plugin@^6.4.0" \
-  eslint-config-standard-with-typescript@latest
+  eslint-config-standard-with-typescript
```

Comment:
Is this change related?

Reply:
If you specify a version explicitly, `yarn add` will put the literal version specified into your `package.json`, unlike npm which will resolve `latest` to a specific patch version range and write that. Leaving the version specifier off will implicitly request `latest`.